packages/lib/src/header/Header.tsx

```diff
@@ -166,34 +23,34 @@ const HeaderContainer = styled.header<{
   margin: HeaderPropsType["margin"];
   underlined: HeaderPropsType["underlined"];
 }>`
+  background-color: var(--color-bg-neutral-lightest);
+  /* TODO: ASK ABOUT UNDERLINE */
+  border-bottom: ${(props) =>
+    props.underlined && `var(--border-width-m) var(--border-style-default) ${props.theme.underlinedColor}`};
```

Comment:
the correct token to be used for the color of the underline is `var(--border-color-neutral-strongest)` as per the design.